test/make.sh

```diff
@@ -18,14 +18,12 @@
 # shebang has a '- e' flag, which causes it
 # to exit on error
 function check_bash() {
-find . -name "*.sh" | while IFS= read -d '' -r file;
-do
-  if [[ "$file" != *"bash -e"* ]];
-  then
-    echo "$file is missing shebang with -e";
-    exit 1;
-  fi;
-done;
+  find . -name "*.sh" | while IFS= read -d '' -r file; do
```

Comment:
Is this related to the change in this PR?

Reply:
No. I think this is my `shfmt` extension deciding to clean the file for me.

Reply:
Reverted.